_commands/tcpdump.md

```diff
@@ -0,0 +1,101 @@
+---
+---
+
+tcpdump
+-------
+
+tcpdump is a tool that allows you to view a dump of traffic on a network. By default, tcpdump will let you view the dump on your local network.
+
+~~~ bash
+$ tcpdump 
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### Example command
+
+tcpdump -S host <hostname>
+
+~~~ bash
+$ sudo tcpdump -S host www.google.com
```

Comment:
Explain why you need to use `sudo`. Also I don't think having so many lines of `tcpdump` makes it any more useful. You can probably cut a bunch out.